schemas/ingest-events.v1.schema.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
To double check: are there other message types sharing this topic right now?

Reply:
Yes! Until we have migrated relay, SDK,  etc to ingest-feedback-events

Reply:
I compared this to the example you added in https://github.com/getsentry/sentry-kafka-schemas/pull/227/.

event_id and type is there, but you have no project_id or payload in your schema? Is that actually true? How do you know what project the user feedback belongs to?

Reply:
I based my fields on this https://develop.sentry.dev/sdk/event-payloads/#required-attributes

Reply:
There is probably project_id in every event, and I can add that. But don't want to make it required, since I don't think it's required for feedback events. (In fact it turns out the feedback fields themselves are all optional 😅)

Reply:
The link you sent represents the payload that the user sends to Relay though, not the one received on this topic which happens after Relay processing. I assume the project ID is automatically added by Relay and is not optional, as a user feedback that doesn't belong to any project doesn't really make any sense.

Reply:
Looking at https://github.com/getsentry/sentry/blob/a1c47440d4399f571fe0161b441811d48148e43c/src/sentry/ingest/consumer/simple_event.py#L42 it seems like a message without a project_id would crash the consumer